src/main/resources/ready-api-rest-api-spec.json

```diff
@@ -327,6 +327,92 @@
       }
     }
   },
+  "/v1/readyapi/executions/{executionID}/logs": {
```

Comment:
Sounds good, How do we merge it back to main version? It should be possible do a diff and copy the changes to main version.